src/preferences/dialog/dlgprefreplaygain.cpp

```diff
@@ -87,26 +110,32 @@ void DlgPrefReplayGain::slotResetToDefaults() {
 
 void DlgPrefReplayGain::slotSetRGEnabled() {
     if (EnableGain->isChecked()) {
-        config->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(1));
+        m_pConfig->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(1));
     } else {
-        config->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(0));
+        m_pConfig->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(0));
     }
     slotUpdate();
     slotApply();
 }
 
-void DlgPrefReplayGain::slotSetRGAnalyzerEnabled() {
+void DlgPrefReplayGain::slotSetRGAnalyzerChanged() {
+    bool rg1 = radioButtonRG1->isChecked();
+    bool disabled = radioButtonDisable->isChecked();
+
     // WARNING: Do not fix the "analyser" spelling here since user config files
     // contain these strings.
-    int enabled = EnableAnalyzer->isChecked() ? 1 : 0;
-    config->set(ConfigKey(kConfigKey,"ReplayGainAnalyserEnabled"),
-                ConfigValue(enabled));
+    m_pConfig->set(ConfigKey(kConfigKey,"ReplayGainAnalyserEnabled"),
+                ConfigValue(!disabled));
+    checkBoxReanalyze->setEnabled(!disabled);
+    m_pConfig->set(ConfigKey(kConfigKey,"ReplayGainVersion"),
```

Comment:
"ReplayGainAnalyserVersion" for consistency? Also applies to the naming of the member functions I suggested above.